tutorhastexo/patches/local-docker-compose-services

```diff
@@ -12,12 +12,14 @@ hastexo-xblock:
     DJANGO_SETTINGS_MODULE: hastexo_guacamole_client.tutor.settings
   restart: unless-stopped
   ports:
-    - 8095:8095
+    - "8085-8095:8095"
```

Comment:
I thought I'd added a comment here earlier, but it looks like that never made it through: could you quickly explain what this is for, and why it doesn't need a corresponding change in the Kubernetes configuration? I'm sure this is fine; I'd just like to understand it. :)

Reply:
As far as I know docker cannot deploy service replicas on same port (should cause `port is already allocated` error), so we'll need to define a port range but we can map the range to the same exposed port on the container.
For Kubernetes, I don't really know how this works, I *think* we don't need to do the same. I was aiming for the trial and error method.

Reply:
Makes sense, thanks!